### ISSUE TITLE
Issue for the first timers only. Create your first PR here. 

### ISSUE THREAD

**Issue by Aniket965** [Hacktoberfest-2018/Hello-world#17]
# Hacktoberfest 2018 Hello-world

# Process 🐬
1. fork the repo
2. Create a folder for a programming language.
3. Add a hello word progam  in the language you like in language folder.
4. if a language folder is already or helloworld is already implemented you can add a programm tha prints your github handle 🙌🏻
5. if you liked sending pr give this repo ⭐️ it will help spreading the good with all.

# finished ❣️

**Comment by jflayhart** [comment]
First timer here :) https://github.com/Hacktoberfest-2018/Hello-world/pull/884

**Comment by chaos4455** [comment]
Please, how i can make a pull request?

**Comment by amohata007** [comment]
https://github.com/amohata007/Hacktoberfest
Visit this link....there are proper steps written already @chaos4455

**Comment by Kirito129** [comment]
#2108 done

**Comment by ginhton** [comment]
#2299 Thank you

**Comment by kichloo** [comment]
Created a Traversal graph in C.Kindly reference it.

**Comment by AmirSoleimani** [comment]
#3744

**Comment by AyushRaj0** [comment]
> Please, how i can make a pull request?

YES. FIRST OPEN ANY REPOSITORY OF SOMEONE AND THEN CLICK ON FORK.
MODIFY THE CONTENT AND CLICK ON PULL REQUEST TAB.
CLICK ON CREATE A PULL REQUEST.

**Comment by kichloo** [comment]
Thanks.

**Comment by vinhyenvodoi98** [comment]
i just fix conflict from vinhyenvodoi98

**Comment by scitronboy** [comment]
@chaos4455 
> Please, how i can make a pull request?
For everyone who wants to create a pull request: checkout [this article](https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/) by Rob Allen.

**Comment by krkic91** [comment]
nice

**Comment by chinela** [comment]
i can still contribute right?

**Comment by scitronboy** [comment]
@chinela sure you can. Why wouldn't you be able to?

**Comment by ancago** [comment]
I'm in! :dancer: so cool
